src/CSS/products.css

```diff
@@ -0,0 +1,623 @@
+.body{
+    width: 320px;
+}
+
+:root{
```

Comment:
Toate culorile care se repeta trebuiesc trecute ca si variabile.
Le gasesti in chatul "Variabile pe discord"